homeassistant/components/totalconnect/config_flow.py

```diff
@@ -65,10 +65,10 @@ async def async_step_locations(self, user_entry=None):
                 if self.usercodes[location_id] is None:
                     valid = await self.hass.async_add_executor_job(
                         self.client.locations[location_id].set_usercode,
-                        user_entry[CONF_LOCATION],
+                        user_entry["usercode"],
```

Comment:
Should you not add in a new const for this instead?

Reply:
updated per suggestion